patches/Avalon/Terraria/GameContent/TextureAssets.cs.patch

```diff
@@ -1,5 +1,35 @@
 --- src/Terraria/Terraria/GameContent/TextureAssets.cs
 +++ src/Avalon/Terraria/GameContent/TextureAssets.cs
+@@ -1,12 +_,11 @@
+ using Microsoft.Xna.Framework.Graphics;
+ using ReLogic.Content;
++using System.Security.RightsManagement;
+ 
+ namespace Terraria.GameContent
+ {
+-	public static class TextureAssets
+-	{
+-		public static class RenderTargets
+-		{
++	public static class TextureAssets {
++		public static class RenderTargets {
+ 			public static PlayerRainbowWingsTextureContent PlayerRainbowWings;
+ 			public static PlayerTitaniumStormBuffTextureContent PlayerTitaniumStormBuff;
+ 			public static PlayerQueenSlimeMountTextureContent QueenSlimeMount;
+@@ -162,6 +_,13 @@
+ 		public static Asset<Texture2D> Chain41;
+ 		public static Asset<Texture2D> Chain42;
+ 		public static Asset<Texture2D> Chain43;
++		public static Asset<Texture2D> AvalonChain38;
++		public static Asset<Texture2D> AvalonChain39;
++		public static Asset<Texture2D> AvalonChain40;
++		public static Asset<Texture2D> AvalonChain41;
++		public static Asset<Texture2D> AvalonChain42;
++		public static Asset<Texture2D> AvalonChain43;
++		public static Asset<Texture2D> AvalonChain44;
```

Comment:
Add fields like these to a partial class instead (I can do this myself manually if you don't want to).